topology/University of Wisconsin–Madison/CHTC/CHTC.yaml

```
@@ -926,7 +926,10 @@ Resources:
           Name: Joe Bartkowiak
         Tertiary:
           ID: OSG1000003
-          Name: Brian Hua Lin
+          Name: Brian Hua Lin        
```

```suggestion
          Name: Brian Hua Lin
```

Remove extraneous whitespace change https://osg-htc.org/technology/software/git-software-development/#avoid-whitespace-noise